src/identity/components/userprofile/UserDefaults.tsx

```diff
@@ -0,0 +1,140 @@
+import {useSelector, useDispatch} from 'react-redux'
+import {
+  Button,
+  ButtonType,
+  ComponentColor,
+  ComponentSize,
+  ComponentStatus,
+  Form,
+} from '@influxdata/clockface'
+
+// Selectors and Context
+import {UserAccountContext} from 'src/accounts/context/userAccount'
+import {selectQuartzIdentity, selectQuartzOrgs} from 'src/identity/selectors'
+
+// Thunks
+import {updateDefaultOrgThunk} from 'src/identity/quartzOrganizations/actions/thunks'
+
+// Components
+import {DefaultAccountForm} from 'src/identity/components/userprofile/DefaultAccountForm'
+import {DefaultOrgForm} from 'src/identity/components/userprofile/DefaultOrgForm'
+
+// Constants
+import {
+  emptyAccount,
+  emptyOrg,
+} from 'src/identity/components/GlobalHeader/DefaultEntities'
+
+// Notifications
+import {notify} from 'src/shared/actions/notifications'
+import {
+  userProfileSaveError,
+  userProfileSaveSuccess,
+} from 'src/shared/copy/notifications'
+
+// Styles
+import 'src/identity/components/userprofile/UserProfile.scss'
+
+export const UserDefaults: FC = () => {
+  const dispatch = useDispatch()
+
+  const {userAccounts, handleSetDefaultAccount} = useContext(UserAccountContext)
+  const quartzOrganizations = useSelector(selectQuartzOrgs)
+
+  const identity = useSelector(selectQuartzIdentity)
+  const loggedInAccount = identity.currentIdentity.account
+
+  const accounts = userAccounts
+  const orgs = quartzOrganizations.orgs
+
+  const defaultAccount = useMemo(
+    () =>
+      accounts ? accounts.find(el => el.isDefault === true) : emptyAccount,
+    [accounts]
+  )
+  const defaultOrg = useMemo(
+    () => (orgs ? orgs.find(el => el.isDefault === true) : emptyOrg),
+    [orgs]
+  )
+
+  const [selectedAccount, setSelectedAccount] = useState(defaultAccount)
+  const [selectedOrg, setSelectedOrg] = useState(defaultOrg)
+
+  useEffect(() => {
+    setSelectedAccount(defaultAccount)
+  }, [accounts, defaultAccount])
+
+  useEffect(() => {
+    setSelectedOrg(defaultOrg)
+  }, [orgs, defaultOrg])
+
+  const selectedNewAccount =
+    defaultAccount?.id !== selectedAccount?.id && selectedAccount !== null
+
+  const selectedNewOrg =
+    defaultOrg?.id !== selectedOrg?.id && selectedOrg !== null
+
+  const userChangedPrefs = selectedNewAccount || selectedNewOrg
+
+  const saveButtonStatus =
+    selectedNewAccount || selectedNewOrg
+      ? ComponentStatus.Default
+      : ComponentStatus.Disabled
+
+  const handleChangeDefaults = async () => {
+    if (userChangedPrefs) {
+      try {
+        if (selectedNewAccount) {
+          await handleSetDefaultAccount(selectedAccount.id, {
+            disablePopUps: true,
+          })
+        }
+
+        if (selectedNewOrg) {
+          await dispatch(
+            updateDefaultOrgThunk({
+              accountId: loggedInAccount.id,
+              newDefaultOrg: selectedOrg,
+            })
+          )
+        }
+
+        dispatch(notify(userProfileSaveSuccess()))
+      } catch (err) {
```

Comment:
Since the user will get the error message from either function failing, there's sort of an atomicity problem: the user could successfully change their account, but then the `orgs` API goes down, and the user would see the error notification. I think that's fine:

1. the new default account will still show correctly - so the user has the correct information. `handleSetDefaultAccount` will update context; which will trigger an update of the `accounts`, which triggers an update of `defaultAccount`, which triggers an update of `setDefaultAccount`.
2. in that case, the user was notified that not everything changed - so they know they have to retry to update their preferences.
3. the atomicity issue would need to be resolved at the quartz level - we can't really do anything about it in the UI, unless we want to _reverse_ the update that was correct (and then what if that reversal fails).